evap/staff/tools.py

```diff
@@ -182,6 +188,9 @@ def bulk_update_users(request, user_file_content, test_run):
         messages.info(request, _('No data was changed in this test run.'))
     else:
         with transaction.atomic():
+            for user in deletable_users + users_to_mark_inactive:
+                # this has to be done in an additional for-loop because otherwise it interferes with the user deletion
+                remove_user_from_represented_and_ccing_users(user, deletable_users + users_to_mark_inactive)
```

Comment:
you don't show any messages here, are you?

Reply:
No, because they would be doubled with the information messages in the previous for-loops.
Should they be shown twice?
The only real problem is the future tense in the messages, but it is also future in all the other notifications. How should it be done?

Reply:
it's fine how it is. i didn't see that the "test" messages are also shown on the real update run :)